develop/toolkit/api-libraries/smart-contracts/ethers-js.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Ethers.js
+description: Learn how to interact with the Asset Hub chain using Ethers.js, compiling and deploying Solidity contracts, and interacting with deployed smart contracts.
+---
+
+# Ethers.js
+
+## Introduction
+
+The `revive` compiler is used to compile Solidity smart contracts to [`PolkaVM`](/develop/smart-contracts/evm/native-evm-contracts/#polkavm){target=\_blank} bytecode, which allows it to be uploaded to Asset Hub. An Ethereum RPC faciliates interaction with existing Ethereum tools, such as Ethers.js or Metamask
```

Comment:
This first paragraph is weird because starts talking about `revive` compiler when the scope of the section is `Ethers`

Also, `An Ethereum RPC faciliates interaction with existing Ethereum tools, such as Ethers.js or Metamask` This should be maybe an introduction in the index page but not in here as we already are explaining what ethers can do